README.md

```diff
@@ -1,117 +1,73 @@
-<p align="center">
-   <img src="https://raw.githubusercontent.com/camelot-dev/camelot/master/docs/_static/camelot.png" width="200">
-</p>
+# Camelot Fork
```

Comment:
that should definitely not go into the `master` branch

Reply:
True that, reverted changes to README.md. If this is getting merged (and that is a big if) we need to update the readme accordingly.